RELEASE/scripts/sl_ascend.ash

```diff
@@ -7387,6 +7374,10 @@ boolean L12_gremlins()
 		}
 	}
 
+	if(0 < have_effect($effect[Curse of the Black Pearl Onion])) {
+		uneffect($effect[Curse of the Black Pearl Onion]);
```

Comment:
What's the deal with this? Something tcrs related?

Reply:
Whoops, forgot to remove from my personal stack of changes. Yeah, it's given by Rhinestones in Sauceror/Blender, makes combat vastly easier, but obviously ruins gremlins. No harm in leaving it in, I suppose.